packages/import-bundle/test/import-bundle.test.js

```diff
@@ -152,6 +161,20 @@ test('inescapable global properties, nested evaluate format', async t => {
   t.is(ns.default, 42);
 });
 
+test('inescapable global properties, script format', async t => {
+  const bundle = await bundleSource(
+    url.fileURLToPath(new URL('export-inescapable-global.js', import.meta.url)),
+    'endoScript',
+  );
+
+  const ns = await importBundle(bundle, {
+    inescapableGlobalProperties: {
+      inescapableGlobalValue: 42,
+    },
+  });
+  t.is(ns.default, 42);
```

Comment:
Not really part of this PR, but should these tests exercise the "inescapability" of `inescapableGlobalProperties` ? Or did we decide that only the lexical-scope additions needed to be inescapable?

Reply:
looks like transforms being inescapable might be important too

Reply:
The test fixture is a module that runs in a Compartment and creates a child Compartment without options. So, all of these are verifying inescapability at least one level down.
```js
export default new Compartment().evaluate('inescapableGlobalValue');
```